.github/workflows/release.yml

```diff
@@ -45,17 +45,16 @@ jobs:
           echo "::set-output name=subdir::$subdir"
```

Comment:
These output vars seem useless, could you check whether we should delete them?

Reply:
You're right, they are redundant.

Reply:
`subdir` is used in `target-path`. `filepath` and `shafilepath` are not used now so delete them